scanner/utils.go

```diff
@@ -211,7 +224,23 @@ func AskForConfig(scanResult models.ScanResultModel) (bitriseModels.BitriseDataM
 	}
 
 	config.App.Environments = append(config.App.Environments, appEnvs...)
+	config.Meta = map[string]interface{}{}
+	config.Meta["bitrise.io.stack"] = selectedStack
```

Comment:
as far as I know, someone requested this to be changed, please make 100% sure this is what we need